app/uk/gov/hmrc/plasticpackagingtax/registration/forms/contact/Address.scala

```diff
@@ -21,13 +21,21 @@ import play.api.data.{Form, Forms, Mapping}
 import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.plasticpackagingtax.registration.forms.CommonFormValidators
 import uk.gov.hmrc.plasticpackagingtax.registration.models.addresslookup.AddressLookupConfirmation
+import uk.gov.voa.play.form.ConditionalMappings.mandatoryIfEqual
+
+case class Country(code: String, name: String)
+
+object Country {
+  implicit val format: OFormat[Country] = Json.format[Country]
+}
 
 case class Address(
   addressLine1: String,
   addressLine2: Option[String] = None,
   addressLine3: Option[String] = None,
   townOrCity: String,
-  postCode: String
+  postCode: Option[String],
+  countryCode: String = "GB"
```

Comment:
I think it's probably sensible to assume a UK address if no country code provided.

Reply:
Although, as seen when 'playing around' with this, this will result in an address obtained (via GRS) from Companies House without country information being flagged as UK based. This might be the best we can hope to do.